modules/nix-bitcoin.nix

```diff
@@ -168,12 +168,7 @@ in {
     services.nix-bitcoin-webindex.enable = cfg.modules == "all";
     services.clightning.autolisten = cfg.modules == "all";
     services.spark-wallet.enable = cfg.modules == "all";
-    services.tor.hiddenServices.spark-wallet = {
-      map = [{
-        port = 80; toPort = 9737;
-      }];
-      version = 3;
-    };
+    services.spark-wallet.onion-service = true;
```

Comment:
Would you consider handling all modules onion services like this from now on? Or would it make the modules less usable for other projects since those might not want to use a hidden service out of the box?

Reply:
> Would you consider handling all modules onion services like this from now on?

Only if necessary as in this case